source/docs/installation.md

```diff
@@ -35,8 +35,4 @@ Install Jigsaw via Composer:
 $ composer require tightenco/jigsaw
 ```
 
-Initialize a new project in the current folder:
```

Comment:
The installation documentation currently shows two different ways to require it: globally and locally. This removes one of those sets of instructions, which we don't want.